polyglot.js

```diff
@@ -469,6 +483,54 @@ function toggleMarkdown({ target: toggle }) {
 }
 
 
+/********************************
+ *     Leaderboard Updates      *
+ ********************************/
+
+function getLeaderboardPositionUrl(lang, user) {
+    return `https://www.codewars.com/api/v1/leaders/ranks/${lang}?user=${user}`;
+}
+
+async function fetchLeaderboardRank(lang, user) {
+    if (!user) user = getUserName();
+    const response = await fetch(getLeaderboardPositionUrl(lang, user));
+    if (response.status === 404) jQuery.notify("User not found!", "error");

```

Comment:
Leaderboards endpoint can handle only 5reqs/min and returns HTTP 429 afterwards. I think it should kick in after changing the trained language a few times. Maybe it would be worth to recognize "Too many requests" and stop when it occurs?